target/linux/ramips/dts/mt7621_raisecom_msg1500-x.00.dts

```diff
@@ -0,0 +1,146 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "raisecom,msg1500-x.00", "mediatek,mt7621-soc";
+	model = "RAISECOM MSG1500 X.00";
+
+	aliases {
+		led-boot = &led_usb;
+		led-failsafe = &led_usb;
+		led-upgrade = &led_usb;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		2.4g {
+			label = "blue:2.4g";
```

Comment:
Most of the other devices use other names like wlan24g, wifi2g, etc. I'd prefer to not introduce a "new name" here, but choose one of the abundant ones ...

Reply:
Thanks for pointing out that.